DesignSafe-CI/portal
# DES-2432: Update Software License

## Overview: ##
Replaced the GNU license with the 3-Clause BSD License

## PR Status: ##


## Related Jira tickets: ##

* [DES-2432](https://jira.tacc.utexas.edu/browse/DES-2432)

## Summary of Changes: ##
In the publication pipeline, the Software License option has been replaced.
Code that references the GNU license has been left for displaying and managing publications that may already be using this license.
New icon file for the new license.

## Testing Steps: ##
1. Take a new project (as in not already published) and go through the pipeline to make sure the new license shows up.
2. On a separate branch, not this branch, do a new publication with the old GNU license.
3. Using this branch, do a new publication with the new 3-Clause license.
4. While still on this branch, go to the publication view and make sure both publications show the appropriate license (gnu or 3-clause).
5. Check the "Download Dataset" window for both publications and check that the appropriate license info is showing based on the publication's license.

## UI Photos:
Showing old license:
<img width="1335" alt="Screenshot 2023-10-10 at 4 29 40 PM" src="https://github.com/DesignSafe-CI/portal/assets/7683802/545164dd-9758-42e8-92dc-abb4663c3574">

Download Dataset for old license:
<img width="1334" alt="Screenshot 2023-10-10 at 4 29 54 PM" src="https://github.com/DesignSafe-CI/portal/assets/7683802/464af4f3-6d50-4ae6-ab04-81f9ce9102fd">

Showing new license:
<img width="1339" alt="Screenshot 2023-10-10 at 4 30 08 PM" src="https://github.com/DesignSafe-CI/portal/assets/7683802/5a5f5ea6-9b00-4126-a90e-3e4ce63d432a">

Download Dataset for new license:
<img width="1348" alt="Screenshot 2023-10-10 at 4 30 18 PM" src="https://github.com/DesignSafe-CI/portal/assets/7683802/d16a9b4d-2735-4a5d-a546-3a0e42f18926">

New license in the publication pipeline:
<img width="1312" alt="Screenshot 2023-10-10 at 4 31 46 PM" src="https://github.com/DesignSafe-CI/portal/assets/7683802/e18e42f7-2586-4295-8c75-ffca14c5db26">

## Notes: ##
